faq/transactional-email.mdx

```diff
@@ -57,4 +57,34 @@ Check out our [dedicated page](https://www.scaleway.com/en/pricing/?tags=managed
 Yes, you can manually unblock any address, even if it is still under the temporary block period. You can unblock up to five addresses every 24 hours.
 
 ## What happens if an address on a TEM blocklist becomes valid again?
-You can manually unblock the address. Make sure you confirm that the initial cause of the bounce was resolved before doing so to avoid further issues.
+You can manually unblock any addresses. Make sure you confirm that the initial cause of the bounce was resolved before doing so to avoid further issues.
```

Comment:
```suggestion
You can manually unblock the address. Before doing so, ensure that the initial cause of the bounce has been resolved to prevent further issues.
```